Makefile

```diff
@@ -48,6 +52,8 @@ GOARCH = $(word 2, $(platform_temp))
 # Set default base image dynamically for each arch
```

Comment:
Could you remove the todo comment on L51?

Reply:
Done